.gitignore

```diff
@@ -4,3 +4,5 @@ bazel-*
 .vscode/
 .DS_Store
 *.swp
+pj.yaml
```

Comment:
Consider storing pj.yaml in /tmp, or allowing the caller of your script determining where if should live.